.travis.yml

```diff
@@ -24,13 +24,15 @@ env:
   - REMOVE_XDEBUG="0"
   - REMOVE_XDEBUG="1"
 
-sudo: false
+sudo: required
 
 cache:
   directories:
     - $HOME/.composer/cache
 
 before_install:
+  - sudo apt-get update --yes
+  - sudo apt-get install --yes net-tools
```

Comment:
@mattrobenolt seems like I [could do it](https://3v4l.org/HFoCC), but I need to find how I can retrieve the port from created socket.